packages/mdc-switch/README.md

```diff
@@ -132,6 +132,9 @@ Mixin | Description
 `mdc-switch-toggled-off-thumb-color($color)` | Sets color of the thumb when the switch is toggled off.
 `mdc-switch-toggled-on-ripple-color($color)` | Sets the color of the ripple surrounding the thumb when the switch is toggled on.
 `mdc-switch-toggled-off-ripple-color($color)` | Sets the color of the ripple surrounding the thumb when the switch is toggled off.
+`mdc-switch-ripple-hover-opacity($opacity)` | Sets the opacity of the ripple surrounding the thumb when hovered.
```

Comment:
We usually do not allow customization based on state. This is usually handled by mdc-states() mixin of mdc-ripple.

What's the context of this change?